### PR TITLE
Fix VS2017 build

### PR DESCRIPTION
**Pull request by schnema123**
Hello, this fixes the build on the newest version of VS2017.
I have also documented a few of the problems I have encountered for others.

Also, I enabled multi-processor compilation on MSVC, that makes a significant difference in build times.

This is my first pull request on github, so be gentle. :P

Greetings.

### CODE REVIEW

File: copying.md [hunk 1]
```diff
@@ -108,6 +108,7 @@ _the openage authors_ are:
 | Julian Guillotel            | arialwhite                  | julian dawt gullotel à gmail dawt com             |
 | Arne Sellmann               | PythonicChemist             | arne dawt sellmann à gmx dawt de                  |
 | Rafael X. Morales Georgi    | chocoladisco                | chocoladisco á gmail dawt com                     |
+| Marcel Schneider            | schnema123                  | marcelschneider5 á outlook dawt de                |
```
Comment (tusharpm) [inline copying.md]: Replace `á` with `à` (similar to line 109 above).
You can optionally fix the last-but-one line also.

File: libopenage/error/handlers.cpp [hunk 2]
```diff
@@ -21,6 +21,7 @@
 #include <unistd.h>
 #endif
 
+#include "../error/handlers.h"
```
Comment (tusharpm) [inline libopenage/error/handlers.cpp]: [The associated header file comes first](https://github.com/SFTtech/openage/blob/11a54bf781c08ff45e93c8c672c8c08d4969b8de/doc/code_style/mom.cpp#L17).
Also, `#include "handlers.h"`.

File: doc/build_instructions/windows_msvc.md [hunk 3]
```diff
@@ -30,6 +31,8 @@
  If you want, you can still use [the prebuilt version](https://www.qt.io/download-open-source/) instead.
  If you do so, include `-DCMAKE_PREFIX_PATH=<QT5 directory>` in the cmake configure command.
 
+ _Note:_ If you are planing to build the 64-bit Version of openage, you are going to need 64-bit libraries. Add command line option `--triplet x64-windows` to the above command or add the enviroment variable `VCPKG_DEFAULT_TRIPLET=x64-windows` to build x64 libraries. [See here](https://github.com/Microsoft/vcpkg/issues/1254)
```
[the remark this answers is not in the document]
Comment (schnema123) [reply doc/build_instructions/windows_msvc.md]: Oops, my bad :)

**Comment by schnema123** [comment]
Ok guys, I had a running build (I have already broken it again.. but I was seeing something that looked like AoE2 on the screen like an hour ago, I swear!). I think I have documented every step that was needed to get there... Feel free to make suggestions.

**Comment by tusharpm** [comment]
Tested it locally. Builds and runs properly for both 32bit and 64bit. Nice work!
@schnema123, whenever you're ready, please group and squash the commits by components (try `git rebase -i master`). Also, while editing the commit messages, prefix the component they modify (`doc`, `buildsystem`, `util`). Some commits like the one modifying `copying.md` might not have prefixes, that's fine.
After this, you'll have to `git push -f` to update the branch on GitHub.
You can remove the `[WIP]` tag from the PR when you are ready for merging.

**Comment by schnema123** [comment]
I have squashed and grouped the commits together. Greetings.

**Comment by tusharpm** [comment]
Thank you, @schnema123! Great work! 👍

**Pull request merged**